pages/login.tsx

```diff
@@ -0,0 +1,107 @@
+import { LockOutlined, UserOutlined } from '@ant-design/icons';
+import { Button, Col, Form, Input, Row } from 'antd';
+import { useForm } from 'antd/lib/form/Form';
+import LoginLayout from 'Components/Layouts/Login/LoginLayout';
+import Image from 'next/image';
+import Link from 'next/link';
+import React from 'react';
+import { P } from 'Styles/Global.styled';
+
+const layout = {
+  labelCol: { span: 8 },
+  wrapperCol: { span: 24 },
+};
+
+export type FormLogin = { email: string; password: string };
+export type FormLoginError = { values: FormLogin; errorFields: [] };
+
+interface Props {
+  onSubmit?: (form: FormLogin) => void;
+  onError?: (form: FormLoginError) => void;
+}
+
+const Login: React.FC<Props> = (props) => {
+  const [form] = useForm<FormLogin>();
+
+  const onFinish = (values) => {
+    props.onSubmit && props.onSubmit(values);
+    console.log('Success:', values);
+  };
+
+  const onError = (form) => {
```

Comment:
risk code, use the shadow name will get mess up

Reply:
how should I name it, do you have any examples?